doc/sphinxext

```diff
@@ -1 +1 @@
-Subproject commit ef988a4a4658c991f4445f6241ab02d74710c6e3
+Subproject commit 84cc897d266e0afc28fc5296edf01afb08005472
```

Comment:
Yeah I was just trying to figure out what is going on here. Not sure why it got changed, but I think I fixed it.